addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/connector/RFXComBaseConnector.java

```diff
@@ -22,7 +24,8 @@
 public abstract class RFXComBaseConnector implements RFXComConnectorInterface {
     private final Logger logger = LoggerFactory.getLogger(RFXComBaseConnector.class);
 
-    private static List<RFXComEventListener> listeners = new ArrayList<RFXComEventListener>();
+    private List<RFXComEventListener> listeners = new ArrayList<>();
```

Comment:
@Jamstah can you agree that this should not be static to properly support multiple bridges?